composer.json

```diff
@@ -8,7 +8,7 @@
     }
   ],
   "require": {
-    "owncloud/ocis-php-sdk": "^1.0.0"
+    "owncloud/ocis-php-sdk": "dev-shareWithMeOcis8016Adjustements"
```

Comment:
to be reverted after https://github.com/owncloud/ocis-php-sdk/pull/166 gets merged